CMakeLists.txt

```
@@ -105,6 +105,17 @@ ENDIF (FORCE_STATIC_PROVIDERS)
 SET (WITH_CORE TRUE CACHE BOOL "Determines whether QGIS core should be built.")
 MARK_AS_ADVANCED(WITH_CORE)
 
+IF (APPLE)
+  # QGIS custom dependencies common location
+  # they can be downloaded from https://qgis.org/downloads/macos/qgis-deps
+  # and unpacked to any location
+  SET (QGIS_MAC_DEPS_DIR "" CACHE PATH "Path to QGIS-Mac-Packager custom dependencies directory")
```

```suggestion
  SET (MAC_DEPS_DIR "" CACHE PATH "Path to QGIS-Mac-Packager custom dependencies directory")
```
so we group everything MAC related at the same place?
or MAC_QGIS_DEPS_DIR ?

---

It is consistent with existing QGIS_MACAPP_* variables we have.

---

let's keep it as is as it's more consistent